Homework3/Task03/task03.py

```diff
@@ -0,0 +1,29 @@
+from collections.abc import Callable
+from typing import Any, List
+
+
+class Filter:
+    """
+    Helper filter class. Accepts a list of single-argument
+    functions that return True if object in list conforms to some criteria
+    """
+
+    def __init__(self, functions: List[Callable]):
+        self.functions = functions
+
+    def apply(self, data: List[Any]) -> List[Any]:
+        return [item for item in data if all(i(item) for i in self.functions)]
+
+
+def make_filter(**keywords: Any) -> Filter:
+    """
+    Generate filter object for specified keywords
```

Comment:
The docstring is a phrase ending in a period. https://www.python.org/dev/peps/pep-0257/

It is time to get docstring plugin back.   https://www.google.com/search?client=firefox-b-d&q=flake8-docstrings

PS. Add other homework folders  to per-file-exclude of flake8.  

Please do it after all other HW will be merged.